tools/abel/model/export_helper.py

```diff
@@ -208,7 +208,7 @@ def _GetConnections(self, entity: Entity) -> Dict[str, List[str]]:
     return {}
 
   def _SortLinks(self, entity: VirtualEntity) -> Dict[str, object]:
-    """Sorts an entity's links by guid and returns a Building Config compliant mapping.
+    """Sorts an entity's links by guid and returns mapping.
```

Comment:
a bc mapping?

Reply:
Added back in but had to change format for line length.